frontend/test/js/spec/user.spec.js

```diff
@@ -68,9 +68,9 @@ define([
                 spyOn(ajax, 'reqwest');
 
                 // PhantomJS double encodes cookies so this enables the getCookie inside of user.js to return the decoded value
-                spyOn(cookie, 'getCookie').and.returnValue(friendUserDetails);
+                spyOn(cookie, 'getDecodedCookie').and.returnValue(friendUserDetails);
 
-                cookie.setCookie(MEM_USER_COOKIE_KEY, friendUserDetails, 1, true);
+                cookie.setCookie(MEM_USER_COOKIE_KEY, window.btoa(JSON.stringify(friendUserDetails)), 1, true);
```

Comment:
This is a bit of a code smell as we no longer set the cookies on the client-side. Having to base64 encode ourselves is not great but is needed for now.  There's probably work to be done to rework these tests to not rely on spys anyway.